api/models/articles.model.js

```diff
@@ -0,0 +1,10 @@
+const db = require("../../db/connection.js");
+
+exports.selectArticleById = (id) => {
+  return db
+    .query(`SELECT * FROM articles WHERE article_id = $1;`, [id])
+    .then(({ rows }) => rows)
+    .catch((err) => {
+      Promise.reject({ status: 500, msg: "Internal Server Error" });
```

Comment:
don't need to check for 500 it will have an automatic PSQL error handler that will do it for you